CHANGELOG.rst

```diff
@@ -1,5 +1,22 @@
 Changelog
 =========
+0.3.0
+`````
+
+Version 0.3.0 updates python-bsonjs's vendored copy of libbson to 1.20.0
+http://mongoc.org/libbson/1.20.0/
+https://github.com/mongodb/mongo-c-driver/blob/1.20.0/src/libbson/NEWS
```

Comment:
I suggest giving some more context for the reader. Like:

> ... vendored copy of libbson to 1.20.0. For a detailed breakdown of what changed in each version of libbson see its changelog:  https://github.com/mongodb/mongo-c-driver/blob/1.20.0/src/libbson/NEWS

Reply:
Done.